internal/sbi/consumer/influenceDataSubscription.go

```diff
@@ -70,9 +73,13 @@ func RemoveInfluenceDataSubscription(ue *pcf_context.UeContext, subscriptionID s
 		logger.ConsumerLog.Warnf("Can't find corresponding UDR with UE[%s]", ue.Supi)
 		return &problemDetail, nil
 	}
+	ctx, pd, err := pcf_context.GetSelf().GetTokenCtx("nnrf-nfm", models.NfType_NRF)
+	if err != nil {
+		return pd, err
+	}
 	udrClient := util.GetNudrClient(ue.UdrUri)
 	httpResp, localErr := udrClient.IndividualInfluenceDataSubscriptionDocumentApi.
```

Comment:
This client is used to communicate with UDR.
The context should be generated by UDR's service and UDR nfType.